static/app/stores/memberListStore.tsx

```diff
@@ -2,56 +2,62 @@ import {createStore, StoreDefinition} from 'reflux';
 
 import {User} from 'sentry/types';
 
+type State = {
+  cursor: string | null;
+  hasMore: boolean | null;
+  loading: boolean;
+  members: User[];
+};
+
 interface MemberListStoreDefinition extends StoreDefinition {
   getAll(): User[];
-  getById(id: string): User | undefined;
-  getState(): User[];
+  getById(memberId: string): User | undefined;
+  getState(): State;
   init(): void;
-  isLoaded(): boolean;
-  loadInitialData(items: User[]): void;
-  loaded: boolean;
-  state: User[];
+  loadInitialData(items: User[], hasMore?: boolean | null, cursor?: string | null): void;
+  state: State;
 }
 
 const storeConfig: MemberListStoreDefinition = {
-  loaded: false,
-  state: [],
+  state: {
+    members: [],
+    loading: true,
+    hasMore: null,
+    cursor: null,
+  },
 
   init() {
     // XXX: Do not use `this.listenTo` in this store. We avoid usage of reflux
     // listeners due to their leaky nature in tests.
 
-    this.state = [];
-    this.loaded = false;
+    this.state = {
+      members: [],
+      loading: true,
+      hasMore: null,
+      cursor: null,
+    };
   },
 
-  // TODO(dcramer): this should actually come from an action of some sorts
-  loadInitialData(items: User[]) {
-    this.state = items;
-    this.loaded = true;
-    this.trigger(this.state, 'initial');
-  },
+  loadInitialData(items: User[], hasMore, cursor) {
+    this.state = {
+      members: items,
+      loading: false,
+      hasMore: hasMore ?? this.state.hasMore,
+      cursor: cursor ?? this.state.cursor,
+    };
 
-  isLoaded() {
-    return this.loaded;
+    this.loading = false;
+    this.hasMore = hasMore ?? this.hasMore;
+    this.cursor = cursor ?? this.cursor;
```

Comment:
Should these be from `this.state` and not `this`?

Reply:
yup, I think I fixed this but didn't push it up 😅